### ISSUE TITLE
Pre-commit doesn't work on Windows

### ISSUE THREAD

**Issue by pluseg** [4xxi/quality-checker#1]
After running pre-commit manually on Windows I see error about `egrep`, message `Good job!` and nothing fixed in code.
```
$ php .git/hooks/pre-commit
The system cannot find the path specified.
'egrep' is not recognized as an internal or external command,
operable program or batch file.
Running "PHPMD"
Running "PHPCS_SYMFONY"
Running "PHPLINT"
Good job!
```

**Comment by mytord** [comment]
#50e57c0

**Comment by mytord** [comment]
Fixed as part of commit #50e57c0
